Doc/library/netrc.rst

```diff
@@ -32,6 +33,10 @@ the Unix :program:`ftp` program and other FTP clients.
 
    .. versionchanged:: 3.4 Added the POSIX permission check.
 
+   .. versionchanged:: 3.7
+      Now uses :func:`os.path.expanduser` to determine the location
+      of ``~/.netrc`` if *file* is not passed as argument.
```

Comment:
`~/.netrc` is Posix-specific syntax. Windows users of `netrc` can even not know what `~/` means. Maybe "of the file :file:'.netrc'" of "of the default netrc file"?